.github/workflows/build_and_push_docker_images.yaml

```diff
@@ -41,9 +44,14 @@ jobs:
           cd source
           tag=${{ github.ref }}
           # tag docker-latest-ce or docker-latest-ee
-          if git describe --tags --exact --match 'docker-latest-*'; then
+          if git describe --tags --exact --match 'docker-latest-*' 2>/dev/null; then
+            echo 'docker_latest=true due to docker-latest-* tag'
```

Comment:
For my own information: is this tagged automatically somehow?  I couldn't find it in any workflows. :thinking:

Reply:
It's not documented anywhere, does not exist in any process checklist and not automated, hence the PR.
I left an option to use `docker-latest*` to override releases if needed, but in general `:latest` should be automatically assigned on new v5.x releases.